yarn.lock

```diff
@@ -2088,7 +2088,7 @@
     levenary "^1.1.1"
     semver "^5.5.0"
 
-"@babel/preset-env@^7.4.5", "@babel/preset-env@^7.8.7":
+"@babel/preset-env@^7.4.5":
```

Comment:
This was erroneously added in #1350 - package.json was updated but yarn.lock wasn't regened.